src/Compilers/CSharp/Test/Symbol/Symbols/GenericConstraintTests.cs

```diff
@@ -5633,13 +5633,13 @@ class B : A
 } 
 ";
             CreateCompilation(source, options: TestOptions.ReleaseDll).VerifyDiagnostics(
-                // (4,21): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' context.
+                // (4,21): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
                 //     public virtual T? Goo<T>()
                 Diagnostic(ErrorCode.WRN_MissingNonNullTypesContextForAnnotation, "?").WithLocation(4, 21),
-                // (4,20): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint.
+                // (4,20): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Please use language version 'preview' or greater to use annotations with unconstrained type parameters, or consider adding a 'class', 'struct', or type constraint.
                 //     public virtual T? Goo<T>()
-                Diagnostic(ErrorCode.ERR_NullableUnconstrainedTypeParameter, "T?").WithLocation(4, 20),
-                // (12,22): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' context.
+                Diagnostic(ErrorCode.ERR_NullableUnconstrainedTypeParameter, "T?").WithArguments("preview").WithLocation(4, 20),
```

Comment:
>WithArguments("preview") [](start = 83, length = 24)

Consider targeting preview version explicitly in order to avoid maintenance when the feature gets moved to CSharp9.